ACMILabs/acmi-api
# #38 Regenerate Works API data

Resolves #38

Regenerate Works API data using the `external=false` filter to remove TMDB, IGDB, and loaned Works.

### Acceptance Criteria
- [x] Regenerate Works API data after [merging external filter branch](https://github.com/ACMILabs/acmi-api/pull/37)
- [x] Check for TMDB records: `grep -ri "TMDB" app/json/`
- [x] Check for IGDB records: `grep -ri "IGDB" app/json/`
- [x] Check for loan records: `grep -r "LN" app/json/`
- [x] Check for loan records: `grep -r "AEO" app/json/`
- [x] Check for loan records: `grep -r "P1" app/json/`

### Relevant design files
* None

### Testing instructions
1. Run the grep commands above over the data and note there are no external works
1. Re-run them on the TSV folder `app/tsv/` and see there are no results

### DoD
For requester to complete:
~- [ ] Deployment / migration instruction have been updated if required~
